src/v3/src/transformer/layout/idp/transformIdvIdpAuthenticator.ts

```diff
@@ -58,7 +94,36 @@ export const transformIdvIdpAuthenticator: IdxStepTransformer = ({
     },
   };
 
-  uischema.elements = [titleElement, descriptionElement, submitButton];
+  const divider: DividerElement = {
+    type: 'Divider',
+  };
```

Comment:
Just curious why only gen3 mockups have a divider?

Reply:
have asked designer. stay tuned :) will do it as follow up if needed.